### PR TITLE
home, contact, market

### PR DESCRIPTION
**Pull request by MarcusWithers**
Still requires minor styling, but just about 100% accurate to design template.

### CODE REVIEW

File: src/css/contact.css [hunk 1]
```diff
@@ -0,0 +1,195 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #fff8e1;
+}
```
Comment (kouYang2015) [inline src/css/contact.css:8]: Move this to theme.css and import.

File: src/css/contact.css [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+.nav-bar li {
+  list-style-type: none;
+  margin-top: 60px;
+  padding: 15px 25px 15px 15px;
+  border-right: solid white;
+}
+
+.nav-bar li:last-child {
+  border-right: none;
+}
```
Comment (kouYang2015) [inline src/css/contact.css:45]: Move these to header.css and import for all pages.

File: src/css/contact.css [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+.footer-contact-text {
+  margin-top: 20px;
+  font-size: 18px;
+}
+
+.footer-contact-text p {
+  margin: 5px 0px 0px 5px;
+}
```
Comment (kouYang2015) [inline src/css/contact.css:83]: move to footer.css and import for all pages.

File: src/css/contact.css [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+  position: relative;
+  top: 5px;
+  left: 850px;
+}
+
+.footer-img p {
+  position: relative;
+  top: 15px;
+  left: 865px;
+}
```
Comment (kouYang2015) [inline src/css/contact.css:108]: Move to footer.css and import to all pages.

File: src/css/contact.css [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+  border-right: solid white;
+}
+
+.nav-bar li:last-child {
+  border-right: none;
+}
+
+h1 {
+  font-size: 72px;
+}
```
Comment (kouYang2015) [inline src/css/contact.css:49]: move to theme.css and import to all pages. Delete from home.css and market.css

File: src/css/contact.css (hunk 1 above)
Comment (kouYang2015) [inline src/css/contact.css:8]: move to theme.css and import to all pages.

File: src/css/home.css [hunk 6]
```diff
@@ -0,0 +1,154 @@
+* {
```
Comment (kouYang2015) [inline src/css/home.css:1]: Find and delete all the redundant classes here that are already defined in theme.css, header.css, and footer.css

File: src/css/market.css [hunk 7]
```diff
@@ -0,0 +1,134 @@
+* {
```
Comment (kouYang2015) [inline src/css/market.css:1]: Find and delete all the redundant classes here that are already defined in theme.css, header.css, and footer.css

**Review by kouYang2015 — CHANGES_REQUESTED** [review]
HTML pages look fine for now. 
Take a look at my comments for the CSS files. There's a lot of redundant classes defined. Let's move them to theme.css, header.css, or footer.css and import them. Keep only classes that are unique to contact, home, and market.css in them.

**Comment by MarcusWithers** [comment]
Sending new pull request with updated css

**Pull request closed without merging**